docs/src/pages/components/tables/EnhancedTable.tsx

```diff
@@ -233,6 +239,17 @@ const useStyles = makeStyles((theme: Theme) =>
     tableWrapper: {
       overflowX: 'auto',
     },
+    visuallyHidden: {
```

Comment:
On a different topic, I was wondering about providing a similar feature: https://getbootstrap.com/docs/4.3/utilities/screen-readers/.

Reply:
I wondered the same thing. (Although Bootstrap's example of using it for a skip-nav is misguided.)